assets/js/base/components/cart-checkout/totals/footer-item/index.js

```diff
@@ -24,11 +24,16 @@ const SHOW_TAXES =
 
 const TotalsFooterItem = ( { currency, values } ) => {
 	const { total_price: totalPrice, total_tax: totalTax } = values;
-	const { extensions } = useStoreCart();
+
+	// Prepare props to pass to the __experimentalApplyCheckoutFilter filter.
+	// We need to pluck out receiveCart.
+	// eslint-disable-next-line no-unused-vars
+	const { extensions, receiveCart, ...cart } = useStoreCart();
 	const label = __experimentalApplyCheckoutFilter( {
 		filterName: 'totalLabel',
 		defaultValue: __( 'Total', 'woo-gutenberg-products-block' ),
 		extensions,
+		arg: { cart },
```

Comment:
There is a data mismatch between this and the data passed in `productPriceValidation` because extensions is plucked out.

Do we need to ensure what we expose to others is consistent across all hooks?